dockerfiles/Dockerfile

```diff
@@ -2,59 +2,53 @@
 
 # BUILD BASE FOR CI
 
-FROM ubuntu:18.04 as base
+FROM ubuntu:20.04 as base
```

Comment:
Right now you can't use a Debian OS for Chipyard since installing the toolchains will break due to dependency conflicts in conda.

Reply:
I'm curious to know more about this.

Reply:
I'm using the following Dockerfile:

```
FROM continuumio/miniconda3
SHELL ["/bin/bash", "-c"]
WORKDIR /root
RUN git clone https://github.com/ucb-bar/chipyard.git && \
    cd chipyard && \
    git checkout conda && \
    conda env create -f scripts/conda-requirements.yaml && \
    conda install -n chipyard -c ucb-bar ucb-bar-riscv-tools
```

I see the following error:

```
UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package libzlib conflicts for:
ucb-bar-riscv-tools -> libcurl[version='>=7.83.1,<8.0a0'] -> libzlib[version='>=1.2.11,<1.3.0a0']
ucb-bar-riscv-tools -> libzlib[version='>=1.2.12,<1.3.0a0']

Package openssl conflicts for:
python=3.8 -> openssl[version='>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1n,<1.1.2a']
ucb-bar-riscv-tools -> libcurl[version='>=7.83.1,<8.0a0'] -> openssl[version='>=1.1.1o,<1.1.2a|>=1.1.1q,<1.1.2a']The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.31=0
  - feature:|@/linux-64::__glibc==2.31=0
  - python=3.8 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.31
```

Reply:
I'll address this later since our main OS's (`centos` + `ubuntu`) are handled.

Reply:
I think that the conda install you pick up from `continuumio/miniconda3` includes the 'defaults' channel and that is contributing tho the solvability issue.  I'd recommend that you use `condaforge/miniforge3` as your base docker image if you want to do something like that.

Reply:
I'll give this a go